### DISCUSSION TITLE
Full Screen Portrait mode Back Button App Close issue

### DISCUSSION THREAD

**Discussion by vikasgoyal087** [vimeo/player.js#985] [Bug reports]
When Video on full screen in portrait mode then user press back button app close issue. How to fix it

**Comment by luwes** [comment]
@vikasgoyal087 could provide some more info? which app exactly and on which device/OS/(browser)? thanks

**Comment by vikasgoyal087** [comment]
Thanks I am working on Ionic 3 app and  this problem on all devices. Video on Landscape mode its working fine.

**Comment by vikasgoyal087** [comment]
Issue not resolved

**Comment by banovotz** [comment]
I am still seeing this issue with Vimeo video being opened from React Native app on android. When user watched video in full screen, portrait mode and goes back in Android button, they are redirected to blank screen with only solution to close the app.

**Comment by bdougherty** [comment]
@vikasgoyal087 @banovotz If this is still an issue, can you provide us with a test case so we can attempt to reproduce?
